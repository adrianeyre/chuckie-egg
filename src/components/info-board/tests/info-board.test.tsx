import React from 'react';
import { shallow } from 'enzyme';

import InfoBoard from '../info-board';
import IInfoBoardProps from '../interfaces/info-board-props';

describe('Info Board', () => {
	it('Should render correctly', () => {
		const defaultProps: IInfoBoardProps = {
			gameOver: true,
			score: 1000,
			startGame: jest.fn(),
		};

		const infoBoard = shallow(<InfoBoard {...defaultProps} />);
		expect(infoBoard).toMatchSnapshot();
	});
});