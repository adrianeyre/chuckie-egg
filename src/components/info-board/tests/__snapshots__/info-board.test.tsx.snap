// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Board Should render correctly 1`] = `ShallowWrapper {}`;
